Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM python:3.7
+
+# Install R system dependencies
+RUN apt-get update \
+&& apt-get install -y r-base r-base-dev libgsl-dev
+
+# Copy workspace files
+WORKDIR /workspace
+COPY ./ ./
+
+# Install R dependencies
+RUN wget -P /tmp \
```

Comment:
Kind of ugly; can be made a tiny bit nicer by having these URLs in a separate text file, but I thought 🤷  how much better is that anyway?